src/translate.js

```diff
@@ -65,6 +65,10 @@ export default function translate(namespaces, options = {}) {
             };
 
             this.i18n.on('initialized', initialized);
+
+            // In case of race condition, that 'initialized' never comes - do immediately 
+            // check ready state + if i18n is initialized.
+            setTimeout(() => !this.state.ready && this.i18n.isInitialized && ready());
```

Comment:
Why it is not sufficient to have the ready() call in the first if block `if (this.i18n.isInitialized) ...`. How can that be not set in the if - but than in the else block? How did you get into that condition...never got that myself.